src/api/spec/controllers/webui/staging/projects_controller_spec.rb

```diff
@@ -29,6 +29,24 @@
       end
     end
 
+    context 'an existent non-staging project' do
+      let!(:existent_project) { create(:project, name: "#{project}:new-staging", maintainer: user) }
+
+      before do
+        post :create, params: { staging_workflow_id: staging_workflow.id, staging_project_name: existent_project.name }
+      end
+
+      subject { staging_workflow }
+
+      it { expect { subject }.not_to change(Project, :count) }
```

Comment:
This would only make sense if `subject` would be the post request. Right now this just tests that creating a staging workflow factory is not creating a new project.

Reply:
True!

To not rewrite all the specs in this file, I'll go with `it { expect(Project.count).to eq(3) }`.